plugins/arDominionB5Plugin/scss/_layout.scss

```diff
@@ -47,3 +47,36 @@
   margin-top: $spacer;
   padding: $spacer;
 }
+
+.atom-slider-for,
+.atom-slider-nav {
+  .slick-track {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  button {
+    &.slick-arrow.slick-prev {
+      left: -30px;
+    }
+
+    &.slick-arrow.slick-next {
+      right: -30px;
+    }
+  }
+}
+
+.atom-slider-nav {
+  img {
+    max-width: 400px;
+    max-height: 300px;
+  }
+}
+
+.atom-slider-for {
+  .slick-slide {
+    display: flex;
+    justify-content: center;
+  }
+}
```

Comment:
I gave it a quick try and this is the final SCSS I got ...

```scss
.atom-carousel {
  .slick-track {
    display: flex;
    align-items: center;

    img {
      max-width: 400px;
      max-height: 300px;
    }
  }

  .slick-arrow.slick-prev {
    left: -30px;
  }

  .slick-arrow.slick-next {
    right: -30px;
  }
}
```

Reply:
Same as what I have after addressing your feedback.  Thanks for your help!